DirectXTex/DirectXTex.h

```diff
@@ -528,6 +528,13 @@ namespace DirectX
         // levels of '0' indicates a full mipchain, otherwise is generates that number of total levels (including the source base image)
         // Defaults to Fant filtering which is equivalent to a box filter
 
+    HRESULT __cdecl CalculateAlphaCoverage(
```

Comment:
I was wondering why you made ``CalculateAlphaCoverage`` and ``ScaleMipMapsAlphaForCoverage`` distinct public functions instead of having just a single function. Based on the usage in ``texconv`` you just call one then the other...

Reply:
No particular reason, other than that's how I structured the problem and solution in my mind. You're right that they're only called in sequence and it's unlikely one would be needed without the other, so I can merge CalculateAlphaCoverage into ScaleMipMapsAlphaForCoverage to clean it up a bit.